build-scripts/bootstrap.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Installing dependencies from brew.."
+# Install tools
+brew reinstall gradle flow watchman xctool
+
+echo "Installing react-native.."
+# Install react-native
+npm install -g react-native-cli
```

Comment:
The tools are used internally by the build for each platform. Running `./gradlew assembleRelease` or the equivalent `xcodebuild` command, fails without these tools available.